stock_predictor/app.py

```diff
@@ -67,19 +69,21 @@ def predict(id: str):
         "enname": "Shanghai Pudong Development Bank Co.,Ltd.",
         "history": [
```

Comment:
I don't quite understand... Is this writing test data in?

Reply:
This is comment. Document comment in Python is wrapped like
```python
"""
This is comment
"""
```